docs/get-started/quickstarts/detect-people.md

```diff
@@ -0,0 +1,108 @@
+images: ["/platform/ml.svg"]
+imageAlt: "Machine Learning"
+---
+
+Follow this guide to detect people with a webcam on the Viam platform, using a [pre trained ML model from the registry](https://app.viam.com/ml-model/ml-models-scuttle/people).
+
+## Requirements
+
+- A computer: a development machine like your laptop and optionally a single-board computer like the Raspberry Pi
+- A webcam: this could be the webcam on your laptop or any other webcam you can connect to your computer
+- An account on the [Viam app](https://app.viam.com)
+
+First, follow the [instructions to create a new machine](/cloud/machines/#add-a-new-machine) representing your computer.
+Follow the {{< glossary_tooltip term_id="setup" text="setup instructions" >}} that appear on your new machine's **CONFIGURE** page to install `viam-server` on your computer and connect it to the Viam app.
+
+You need to configure the following {{< glossary_tooltip term_id="resource" text="resources" >}} to detect people with a webcam:
+
+1. A [`webcam` camera](/components/camera/webcam/)
+2. An [`tflite_cpu` ML model service](/services/ml/deploy/tflite_cpu/)
+3. An [`mlmodel` computer vision service](/services/vision/mlmodel/)
+
+Follow these instructions to configure your machine and test detections:
+
+{{%expand "Step 1: Configure your webcam" %}}
+
+Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
+Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
+Select the `camera` type, then select the `webcam` model.
+Enter a name or use the suggested name for your camera and click **Create**.
+
+Click the **Save** button in the top right corner of the page to save your config.
+Go to the **CONTROL** tab and expand the camera's remote control card to test the camera stream.
+If you don't see an image stream, you need to [configure the `video_path` attribute](/components/camera/webcam/#using-video_path).
+
+For more detailed information, including optional attribute configuration, see the [`webcam` docs](/components/camera/webcam/).
+
+{{% /expand%}}
+{{%expand "Step 2: Deploy a person detection model" %}}
+
+Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
+Click the **+** icon next to your machine part in the left-hand menu and select **Service**.
+Select the `ML model` type, then select the `TFLite CPU` model.
+Enter a name or use the suggested name for your service and click **Create**.
+
+To deploy a person detector model from the registry, select **Deploy model on machine** for the **Deployment** field in the resulting vision service configuration pane.
+
+Click on **Select models** to open a dropdown with all of the ML models available to you.
+Select the **people** model by **ml-models-scuttle** to deploy an object detection TFLite model that's pre trained with person detection.
```

Comment:
Supernit: optionally include the exact string we expect to see, since we are otherwise presenting the two parts in opposite order that they appear (for left-right language readers): `ml-models-scuttle:people`.

Maybe:

```suggestion
Select the `ml-models-scuttle:people` model (the **people** model by **ml-models-scuttle**) to deploy an object detection TFLite model that has been trained to be able to detect a person.
```